Unable to execute kaprino on a docker image due to ld not installed

This is a log which represents the problem.
```
[LOG] Kaprino started
[LOG] Read input files succeeded "test.kpr"
[LOG] This program contains 1statement(s)
[LOG] Parsing succeeded
[LOG] Defined Text
[LOG] Defined B
[LOG] Defined R
[LOG] Defined Z
[LOG] Optimization: Default
[LOG] Execute external tool: clang -o test test.o
clang-9: error: unable to execute command: Executable "ld" doesn't exist!
clang-9: error: linker command failed with exit code 1 (use -v to see invocation)
[ERROR] Clang faield: 256
[LOG] Kaprino returns: 256
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -101,7 +101,7 @@ RUN \
     apk add --no-cache --virtual builddep \
         wget; \
     apk add --no-cache \
-        clang \
-        python3; \
+        binutils \
+        clang; \
     apk del builddep; \
     rm -rf /var/cache/apk/*;

